.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: main
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+
+      - name: Build a project & run unit tests
```

Comment:
To jeszcze jedna rzecz - `run unit tests` jest nie do końca prawdą, bo testy mamy głównie integracyjne. Może po prostu `run tests`?

Reply:
jasne, zmieniłem :)